__device-tests__/helpers/caps.js

```diff
@@ -0,0 +1,25 @@
+exports.ios12 = {
+	browserName: '',
+	platformName: 'iOS',
+	platformVersion: '12.0',
+	deviceName: 'iPhone Simulator',
+	os: 'iOS',
+	deviceOrientation: 'portrait',
+	automationName: 'XCUITest',
+	appiumVersion: '1.9.1',
+	app: undefined, // will be set later
+};
+
+exports.android8 = {
```

Comment:
I'm not familiar with how Appium works but I tried running the tests on my device and had the following error.
```
The environment you requested was unavailable.
    An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 8.0. The following are available: NB1GAS3772217065 (9)
```
Is it possible to add support for android 9 as well?

Reply:
RIght, the reason there's android 8 here is because the CI solution we're using for emulators(Sauce labs) currently hasn't released support for Android 9. I'll check for a timeline on that. Alternatively locally you add another configuration for Android 9 and change out https://github.com/wordpress-mobile/gutenberg-mobile/blob/628bb9e8a61eb01392466ebdbc86c8f01f942830/__device-tests__/helpers/utils.js#L57. In the future I'd like to make it easier to change configurations. 

The error itself indicates you don't have an Android 8 emulator running, you could try firing up one of those or changing the config that I mentioned above.